CMakeLists.txt

```diff
@@ -1,6 +1,27 @@
 cmake_minimum_required(VERSION 3.18)
 project(tvm C CXX)
 
+MESSAGE(STATUS "c compiler :  ${CMAKE_COMPILER_IS_GNUCC}")
+message(STATUS "system name : ${CMAKE_SYSTEM_NAME}")
```

Comment:
thanks for the PR! just wanted to point out that we have `Summary.cmake`: https://github.com/apache/tvm/blob/main/cmake/utils/Summary.cmake#L47 (which you can enable with `-DSUMMARIZE=1` or `set(SUMMARIZE ON)` in `config.cmake`), so these info messages could be moved there